timeseries/src/autogluon/timeseries/models/autogluon_tabular/mlforecast.py

```diff
@@ -12,8 +12,12 @@
 from autogluon.tabular import TabularPredictor
 from autogluon.timeseries.dataset.ts_dataframe import ITEMID, TIMESTAMP, TimeSeriesDataFrame
 from autogluon.timeseries.models.abstract import AbstractTimeSeriesModel
+from autogluon.timeseries.utils.datetime import (
+    get_lags_for_frequency,
```

Comment:
Do we need to define additional functions at AG, considering that gluonts already has similar ones (e.g., this [one](https://github.com/awslabs/gluonts/blob/0bc0a459263d1599f5f8eaa6d5e94647917fcef4/src/gluonts/time_feature/lag.py#L29))?

Reply:
We used to rely on these methods from GluonTS, but they do not support many of the pandas frequencies (e.g., C, BM, CBM, CBMS, SMS, SM, BQ, BQS, BH, L, U, N). Our implementation of these methods is equivalent to the GluonTS implementations for the frequencies that GluonTS supports, but we also add support for the other pandas frequencies where GluonTS fails.